packages/ove-app-charts/README.md

```diff
@@ -0,0 +1,55 @@
+# Charts App
+
+This app supports visualisation of charts (excluding network diagrams) using the OVE framework. It is based on [Vega](https://vega.github.io/vega/), a declarative language for creating, saving, and sharing interactive visualization designs. Vega supports Canvas and SVG rendering.
+
+## Application State
+
+The state of this app has a format similar to what is provided below:
+
+```json
+{
+    "spec": "https://raw.githubusercontent.com/vega/vega/master/docs/examples/bar-chart.vg.json",
+    "options": {
+        "width": 800,
+        "height": 800
+    }
+}
+```
+
+The `spec` field points to either a URL of, or contains an embedded [Vega specification](https://vega.github.io/vega/docs/#specification).
+
+## Loading the App
+
+A chart can be loaded using the OVE APIs:
+
+Windows:
+
+```sh
+curl --header "Content-Type: application/json" --request POST --data "{\"h\": 500, \"app\": {\"url\": \"http://OVE_APP_CHARTS_HOST:PORT\", \"states\": {\"load\": {\"spec\": \"https://raw.githubusercontent.com/vega/vega/master/docs/examples/bar-chart.vg.json\", \"options\": {\"width\": 800, \"height\": 800}}}}, \"space\": \"LocalNine\", \"w\": 500, \"y\": 0, \"x\": 0}" http://OVE_CORE_HOST:PORT/section
+```
+
+Linux/Mac:
+
+```sh
+curl --header "Content-Type: application/json" --request POST --data '{"h": 500, "app": {"url": "http://OVE_APP_CHARTS_HOST:PORT","states": {"load": {"spec": "https://raw.githubusercontent.com/vega/vega/master/docs/examples/bar-chart.vg.json", "options": {"width": 800, "height": 800}}}}, "space": "LocalNine", "w": 500, "y": 0, "x": 0}' http://OVE_CORE_HOST:PORT/section
+```
+
+The charts app has a transparent background. If required, a background colour of choice can be set using the OVE APIs:
+
+Windows:
+
+```sh
+curl --header "Content-Type: application/json" --request POST --data "{\"h\": 500, \"app\": {\"url\": \"http://OVE_APP_HTML_HOST:PORT\", \"states\": {\"load\": {\"url\": \"/data/background/index.html?background=COLOUR\"}}}, \"space\": \"LocalNine\", \"w\": 500, \"y\": 0, \"x\": 0}" http://OVE_CORE_HOST:PORT/section
+```
+
+Linux/Mac:
+
+```sh
+curl --header "Content-Type: application/json" --request POST --data '{"h": 500, "app": {"url": "http://OVE_APP_HTML_HOST:PORT","states": {"load": {"url": "/data/background/index.html?background=COLOUR"}}}, "space": "LocalNine", "w": 500, "y": 0, "x": 0}' http://OVE_CORE_HOST:PORT/section
```

Comment:
This is not really a documentation question, but why does is the background color a parameter within the ``url``, rather than being a separate attribute in the ``state`` object?

Reply:
A state object is defined by OVE, the background is provided by a separate utility. I saw no reason to couple them.

Reply:
This is being discussed elsewhere - so I am resolving.